magenta-lib/src/main/scala/magenta/tasks/gcp/GCSTasks.scala

```diff
@@ -49,7 +53,7 @@ case class GCSUpload(
   def fileString(quantity: Int) = s"$quantity file${if (quantity != 1) "s" else ""}"
 
   // end-user friendly description of this task
-  def description: String = s"Upload ${fileString(objectMappings.size)} to GCS bucket $bucket using file mapping $paths"
+  def description: String = s"Upload ${fileString(objectMappings.size)} to GCS bucket $gcsTargetBucket using file mapping $paths"
```

Comment:
Should this include the .name

Reply:
It's a case class so it has a toString method which enables the whole bucket configuration to appear in the riffraff deploy output: Which is useful. I might make this more explicit:
But for example: `[10:21:48] task GCSUpload Upload 163 files to GCS bucket GcsTargetBucket(europe-west2-datatech-airfl-fdf1e4b5-bucket,List(dags/dags, dags/guardian),List(.py, .json)) using file mapping List((S3Path(riffraff-artifact,ophan:data-lake:airflow/7375/cloud-composer-deployment/),))`